drivers/can/can_mcp2515.c

```diff
@@ -352,6 +371,25 @@ static int mcp2515_configure(struct device *dev, enum can_mode mode,
 		goto done;
 	}
 
+	/* Wait 128 OSC1 clock cycles at 1MHz (minimum clock in frequency)
+	 * see MCP2515 datasheet section 8.1 Oscillator Start-up Timer
+	 */
+	k_usleep(128);
```

Comment:
It's probably worth putting another 128us sleep before the soft reset command.  I don't know how quickly the driver could be initialised after a system is released from reset but if it's quicker than 128us then the soft reset command may be blocked due to the same 128 OSC1 clock cycle requirement.

Reply:
Thoughts @karstenkoenig ?

Reply:
Guess waiting another 128us is safer than not waiting. I won't object nor push for it though, as technically this implies a perfect release from reset between both MCU and MCP2515.

Reply:
At the moment there will be somewhere between 0us to >128us between the MCP2515 release from reset or power up.  Adding a 128us sleep before the soft reset will just ensure that 128us is the minimum.  @NZSmartie I wont block the PR either way you choose to go.  Thanks for your fix.